<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::trackviewdialog</class>
 <widget class="QDialog" name="snowgui::trackviewdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="buttonWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="offsetPxButton">
        <property name="text">
         <string>offset Px</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="offsetArcsecButton">
        <property name="text">
         <string>offset arc sec</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="correctionButton">
        <property name="text">
         <string>correction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="snowgui::ChannelDisplayWidget" name="gpWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::ChannelDisplayWidget</class>
   <extends>QWidget</extends>
   <header location="global">ChannelDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
